bindings/context_bind.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef INCLUDED_GHEX_FORTRAN_CONTEXT_BIND_HPP
+#define INCLUDED_GHEX_FORTRAN_CONTEXT_BIND_HPP
+
+#include "obj_wrapper.hpp"
+#include <vector>
+#include <ghex/transport_layer/util/barrier.hpp>
+
+namespace ghex = gridtools::ghex;
+
+#ifdef GHEX_USE_UCP
+
+/* UCX backend */
+#include <ghex/transport_layer/ucx/context.hpp>
+using transport    = ghex::tl::ucx_tag;
```

Comment:
should we move those types into a namespace as well?

Reply:
darn! I still left out some! will do